### ISSUE TITLE
Installation instructions for Fedora

### ISSUE THREAD

**Issue by ldorau** [xflux-gui/fluxgui#95]
I cannot find any installation instructions for Fedora OS.
Could you add them?

**Comment by ntc2** [comment]
I don't have a Fedora system, but assuming there is not a Fedora PPA, it should be easy to adapt the [Debian/Ubuntu instructions](https://github.com/xflux-gui/fluxgui#ubuntudebian-manual-install). You just need to figure out the analog of

    sudo apt-get install git python-appindicator python-xdg python-pexpect python-gconf python-gtk2 python-glade2 libxxf86vm1

on Fedora, using `dnf` or whatever the package manager is these days. The rest of the instructions will be the same / are independent of your Linux distribution.

Once you figure out how to install the dependencies on Fedora, it would be helpful if you submitted a PR on the README, or commented here explaining what you did.

**Comment by ldorau** [comment]
Indeed, the installation procedure is quite the same. I could submit a PR, but I have two problems now:
1) ```python download-xflux.py``` does not work for me:
```
$ python download-xflux.py 
Downloading 64-bit xflux ...
Extracting /tmp/xflux.tgz ...
Traceback (most recent call last):
  File "download-xflux.py", line 34, in <module>
    download_xflux()
  File "download-xflux.py", line 29, in download_xflux
    tar = tarfile.open(tarchive)
  File "/usr/lib64/python2.7/tarfile.py", line 1680, in open
    raise ReadError("file could not be opened successfully")
tarfile.ReadError: file could not be opened successfully
```
The downloaded file '/tmp/xflux.tgz' is in fact a html page:
```
Request Error (invalid_request)
--
Your request could not be processed. Request could not be handled
This could be caused by a misconfiguration, or possibly a malformed request.
For assistance, contact your network support team.
```

2) I am experiencing the common issue https://github.com/xflux-gui/xflux-gui/issues/27:
```Sorry, we only support 8 and 10-bit displays right now.```

So I could submit a PR, but without the ```python download-xflux.py``` stage.

**Comment by ldorau** [comment]
Without downloadingf xflux it could look like [this](https://github.com/ldorau/fluxgui/tree/Add-manual-installation-instructions-for-Fedora#fedora-manual-install).

**Comment by faraco** [comment]
@ldorau 

```
Request Error (invalid_request)
--
Your request could not be processed. Request could not be handled
This could be caused by a misconfiguration, or possibly a malformed request.
For assistance, contact your network support team.
```

My guess for the above error is due to your networking settings.

I'm not entirely sure if my last commit for the `download-xflux.py` caused your 1. issue but for now, you can try the old code here (using your system `wget` and `tar` instead of Python library utils).

[old-download-xflux.py](https://raw.githubusercontent.com/xflux-gui/fluxgui/5e27b6166a205239b5d075f5568ca5edf86051cb/download-xflux.py)

**Comment by ldorau** [comment]
@faraco the [old version](https://raw.githubusercontent.com/xflux-gui/fluxgui/5e27b6166a205239b5d075f5568ca5edf86051cb/download-xflux.py) of ```download-xflux.py``` works well for me, so the commit https://github.com/xflux-gui/fluxgui/commit/cd0773605c8b91527c039945481856645463c064 can be the cause of my problem.

**Comment by ntc2** [comment]
This was just fixed by PR #98.
